srv/modules/modules/difference.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
whats the reason behind having this module in `srv/modules/modules/difference.py` and in `srv/salt/_modules/difference.py ` ?

Reply:
https://github.com/SUSE/DeepSea/pull/1783/files#diff-906fd9295174e9bb5a7ee5ea4b8a6f5fR5